.github/workflows/build-linux.yml

```diff
@@ -52,29 +52,6 @@ jobs:
       - name: Test docs
         run: ./scripts/gear.sh test docs
 
-      - name: Build docs
```

Comment:
`cargo test --doc` might not catch all of documentation problems so I'd revert this step